### ISSUE TITLE
Enhancements to imshow

### ISSUE THREAD

**Issue by hansenms** [ismrmrd/ismrmrd-python-tools#3]
We should add:

* Colorbar
* Some way of looking at pixels

**Comment by inati** [comment]
What do you mean by looking at pixels?  Do you mean clicking on a point to get its value?

**Comment by hansenms** [comment]
@inati, yes, something like that. Ability to plot a like would be great too and drawing a circle or something.

**Comment by hansenms** [comment]
I added an option to get a colorbar.

**Comment by naegelejd** [comment]
I wrote a simple event handler for click events and I can print the value
of the selected point to the terminal, but I'm not sure how useful that is.
Would you rather it pop up in a message box, appear dynamically in the
original plot window, or something else?

--
Joe Naegele
joseph.naegele@gmail.com
josephnaegele.com
301.676.1403

**Comment by hansenms** [comment]
I will take a look at what you did, but I think we should probably sit down and discuss what functionality we would like and how hard it is to do.

**Comment by grlee77** [comment]
here are a couple resources of possible interest:

For interactive 2D plotting and the ability to draw lines for plotting line profiles, etc. there is some nice code in the `viewer` module of [scikit.image](http://scikit-image.org/).  specific example:
https://github.com/scikit-image/scikit-image/blob/master/viewer_examples/plugins/lineprofile.py
The viewer displays coordinates and corresponding pixels values as well.

There is also a pretty good interactive orthogonal slice viewer for 3D/4D volumes proposed in this PR for nibabel:
https://github.com/nipy/nibabel/pull/251
